integrations/amazon_bedrock/tests/test_chat_generator.py

```diff
@@ -1,37 +1,42 @@
 import json
-import logging
-import os
-from typing import Optional, Type
-from unittest.mock import MagicMock, patch
 
 import pytest
 from haystack.components.generators.utils import print_streaming_chunk
 from haystack.dataclasses import ChatMessage, ChatRole, StreamingChunk
 
 from haystack_integrations.components.generators.amazon_bedrock import AmazonBedrockChatGenerator
-from haystack_integrations.components.generators.amazon_bedrock.chat.adapters import (
-    AnthropicClaudeChatAdapter,
-    BedrockModelChatAdapter,
-    MetaLlama2ChatAdapter,
-    MistralChatAdapter,
-)
 
 KLASS = "haystack_integrations.components.generators.amazon_bedrock.chat.chat_generator.AmazonBedrockChatGenerator"
-MODELS_TO_TEST = ["anthropic.claude-3-sonnet-20240229-v1:0", "anthropic.claude-v2:1"]
-MODELS_TO_TEST_WITH_TOOLS = ["anthropic.claude-3-haiku-20240307-v1:0"]
-MISTRAL_MODELS = [
-    "mistral.mistral-7b-instruct-v0:2",
-    "mistral.mixtral-8x7b-instruct-v0:1",
+MODELS_TO_TEST = [
+    "anthropic.claude-3-5-sonnet-20240620-v1:0",
+    "cohere.command-r-plus-v1:0",
     "mistral.mistral-large-2402-v1:0",
 ]
+MODELS_TO_TEST_WITH_TOOLS = [
+    "anthropic.claude-3-5-sonnet-20240620-v1:0",
+    "cohere.command-r-plus-v1:0",
+    "mistral.mistral-large-2402-v1:0",
+]
+
+# so far we've discovered these models support streaming and tool use
+STREAMING_TOOL_MODELS = ["anthropic.claude-3-5-sonnet-20240620-v1:0", "cohere.command-r-plus-v1:0"]
+
```

Comment:
These models in MODELS_TO_TEST, MODELS_TO_TEST_WITH_TOOLS and STREAMING_TOOL_MODELS @askainet 🙏 🙏

Reply:
Paul set us up 🚀